lazyflow/operators/ioOperators/opInputDataReader.py

```diff
@@ -67,7 +65,9 @@
 import re
 import logging
 
-from lazyflow.utility.io_util.multiprocessHdf5File import MultiProcessHdf5File
+from lazyflow.utility.io_util.multiprocessHdf5File import \
+    MultiProcessHdf5File
```

Comment:
I'm puzzled about this one, what is your line length limit? I think we have agreed to 120.

Reply:
this was just an exemplary comment, there are a lot of these changes